test/models/root_repository_test.rb

```diff
@@ -765,5 +764,24 @@ def test_audit_on_repo_destroy
       assert_equal 'Katello::RootRepository', recent_audit.auditable_type
       assert_equal 'destroy', recent_audit.action
     end
+
+    def test_audit_hook_to_find_records_should_return_content
+      @fedora_root.save!
+      content_id = 'dummycontent-123'
+      content = FactoryBot.create(:katello_content, cp_content_id: content_id, :organization_id => @product.organization_id)
+      FactoryBot.create(:katello_product_content, content: content, product: @product)
+      @fedora_root.update_attributes!(content_id: content_id)
+      @audit_record = @fedora_root.audits.where(:action => 'update').first
+      refute_empty @audit_record.audited_changes['content_id']
+      assert_nil Katello::RootRepository.reflect_on_association('content')
+
+      if Katello::RootRepository.respond_to?('audit_hook_to_find_records')
```

Comment:
Any reason to have this `respond_to?` line? I wouldn't think it is needed since its deliberately added to the model, it seems like a safe assumption to make.

Reply:
Perhaps a compatibility layer for older foreman without this new hook (1.24 addition I think)